index.html

```diff
@@ -0,0 +1,318 @@
+                                <li class="skill-card-tech-tag capybara"><p class="skill-card-tech-tag-text">Capybara</p></li>
+                                <li class="skill-card-tech-tag selenium"><p class="skill-card-tech-tag-text">Selenium</p></li>
+                            </div>
+                        </ul>
+                    </div>
+                </section>
+                <!-- skills -->
+                <section class="skill-card">
+                    <!-- icon -->
+                    <div class="skill-card-icon">
+                        <div class="skill-card-icon-container"><img class="skill-card-icon-img" src="img/skill/skills-img.svg" alt=""></div>
+                    </div>
+                    <!-- title and tags -->
+                    <div  class="skill-card-content">
+                        <h3 class="skill-card-title">Skills</h3>
+                        <ul class="skill-card-rows">
+                            <div class="skill-card-tech">
+                                <li class="skill-card-tech-tag codekit"><p class="skill-card-tech-tag-text">Codekit</p></li>
+                                <li class="skill-card-tech-tag github"><p class="skill-card-tech-tag-text">GitHub</p></li>
+                                <li class="skill-card-tech-tag codepen"><p class="skill-card-tech-tag-text">Codepen</p></li>
+                                <li class="skill-card-tech-tag gitlab"><p class="skill-card-tech-tag-text">Gitlab</p></li>
+                            </div>
+                            <div class="skill-card-tech">
+                                <li class="skill-card-tech-tag terminal"><p class="skill-card-tech-tag-text">Terminal</p></li>
+                            </div>
+                        </ul>
+                    </div>
+                </section>
+            </div>
+        </section>
+        
+        <!-- Form Section -->
+        <section id="form">
+            <!-- text -->
+            <div id="form-text">
+                <p id="form-text-item">I'm always interested in hearing about new projects, so if you'd like to chat please get in touch.</p>
+            </div>
+            <!-- form -->
+            <form action="https://formspree.io/f/myyawpqy" method="POST" id="form-content">
+                <div class="form-row" id="first-form-row">
+                    <input type="text" required name="first-name" placeholder="First Name" class="form-row-column">
+                    <input type="text" required name="last-name" placeholder="Last Name" class="form-row-column">
+                </div>
+                <input type="text" required name="full name" placeholder="Full Name" class="form-row" id="ignoreindesktop">
+                <input type="email" required name="email" placeholder="Email Address" id="" class="form-row">
+                <textarea name="message" id="" cols="30" rows="10" placeholder="Typing..." class="form-row textareas"></textarea>
```

Comment:
- [ ] Please, ensure that the `form-text` is aligned and centered to the `form` element like it is in the design.

- Your design 
<img width="807" alt="wrong1" src="https://user-images.githubusercontent.com/41163571/220451316-0f540d11-0ee5-4e58-b4a9-6a4a21d609b7.png">

- Figma design
<img width="330" alt="right1" src="https://user-images.githubusercontent.com/41163571/220451378-15082901-aaa9-4424-be9a-bfce888fd0ae.png">